apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -494,17 +512,29 @@ export class ExtractorConfig {
 
       if (!configObject.mainEntryPointFile) {
         // A merged configuration should have this
-        throw new Error('mainEntryPointFile is missing');
+        throw new Error('The "mainEntryPointFile" setting is missing');
       }
       const mainEntryPointFile: string = ExtractorConfig._resolvePathWithTokens('mainEntryPointFile',
         configObject.mainEntryPointFile, tokenContext);
 
       if (!ExtractorConfig.hasDtsFileExtension(mainEntryPointFile)) {
-        throw new Error('The mainEntryPointFile is not a declaration file: ' + mainEntryPointFile);
+        throw new Error('The "mainEntryPointFile" value is not a declaration file: ' + mainEntryPointFile);
       }
 
       if (!FileSystem.exists(mainEntryPointFile)) {
-        throw new Error('The mainEntryPointFile does not exist: ' + mainEntryPointFile);
+        throw new Error('The "mainEntryPointFile" path does not exist: ' + mainEntryPointFile);
+      }
+
+      const tsconfigFilePath: string = ExtractorConfig._resolvePathWithTokens('tsconfigFilePath',
+        configObject.compiler.tsconfigFilePath, tokenContext);
+
+      if (configObject.compiler.overrideTsconfig === undefined) {
+        if (!tsconfigFilePath) {
+          throw new Error('Either the "tsconfigFilePath" or "overrideTsconfig" setting must be specified');
+        }
+        if (!FileSystem.exists(tsconfigFilePath)) {
+          throw new Error('The "tsconfigFilePath" path does not exist: ' + tsconfigFilePath);
```

Comment:
*The file referenced by "tsconfigFilePath" does not exist: ...

Reply:
Fixed